lib/src/commands/Deprecations.ts

```diff
@@ -1,10 +1,22 @@
+import isEqual from 'lodash/isEqual';
+import once from 'lodash/once';
+import { Platform } from 'react-native';
 
 export class Deprecations {
-  public onProcessOptions(_key: string, _parentOptions: Record<string, any>) {
-
+  public onProcessOptions(key: string, parentOptions: Record<string, any>, commandName: string) {
+    if (
```

Comment:
This will warn each time visible is defined in options - even if it doesn't actually trigger a toggle which can be very confusing.
What about triggering the console.warn from the animate/toggle method in native? We can emit a "DeprecationEvent" from native to js, handle the event by registering an event listener in this class and console warning in the event handler.